monitoring/src/check-page/aus.ts

```diff
@@ -0,0 +1,204 @@
+import type { Browser, Page } from 'puppeteer-core';
+import {
+	checkCMPIsNotVisible,
+	checkCMPIsOnPage,
+	checkTopAdHasLoaded,
+	clearCookies,
+	loadPage,
+	log_error,
+	log_info,
+	makeNewBrowser,
+} from './common-functions';
+
+// -- Original code from Robert ------------------------------------------------
```

Comment:
I think we should remove the commented out code here referencing the other implementations.